TenureInformationApi/V1/Boundary/Requests/UpdateTenureRequestObject.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using TenureInformationApi.V1.Domain;
+
+namespace TenureInformationApi.V1.Boundary.Requests
+{
+    public class UpdateTenureRequestObject
+    {
+        public Guid Id { get; set; }
```

Comment:
Is this ID needed? 

It looks like you are getting the tenure by the id passed in as a query parameter on `TenureQueryRequest` but you are updating the tenure by the ID from this model?

Also, this is the request that comes in from the API call, but it is then used inside the Gateway which looks incorrect?

Reply:
Some more commentary: I would suggest taking a look at how PersonApi deals with Update - it would be good to keep the implementation consistent with that?

Reply:
The user is not required to provide us with a tenure ID in the request object, the gateway auto-detect the tenureid as the id given in the query.  We need this id  in the request object so dynamodb knows which tenure to update

Reply:
Also, the PersonAPI update endpoint works slightly different as it has a version number and if-match to detect which exact values we want to implement, is that how we want to do this endpoint (I personally don't think that'll be useful as we only have the household member object that can be updated)